views/components/code_example/_code_example.scss

```diff
@@ -35,8 +35,15 @@
 	.output pre {
 		@extend %custom-prism-light-mode;		
 	}
+}
+figure {
```

Comment:
I'm generally wary of global (unscoped) rule sets like this, particularly if they reside within a component's style sheet. No need to address this immediately, just something to keep in mind.

Reply:
Yes, that is a very good point. I actually had the same thought, but I knew that this is the only instance of figure currently in the website. I'll probably get mad at myself though, if I decide to add another one in a different context.

After sleeping over it, I am going to try and see if I can add the `code-example` class to the `figure` itself (which would be cleaner markup as well), and maybe that could clean things up a bit.

Reply:
FWIW (really just an inconsequential observation), `code-sample` scans better for me; makes for a better rhythm.

Reply:
🤔 I could get behind that 🤔 

Naming is usually what is most difficult for me.